airflow/www/views.py

```diff
@@ -1095,107 +1096,22 @@ def success(self):
         recursive = request.args.get('recursive') == "true"
```

Comment:
Do you want to remove the `recursive` option? I think you have mentioned it somewhere. Not only here, but also from the html and js, /www/templates/airflow/dag.html